backend/kesaseteli/common/tests/factories.py

```diff
@@ -415,6 +415,69 @@ class YouthApplicationFactory(AbstractYouthApplicationFactory):
     status = factory.Faker("random_element", elements=YouthApplicationStatus.values)
 
 
+class AbstractNonVtjYouthApplicationFactory(AbstractYouthApplicationFactory):
+    """
+    An abstract base class for youth applications created using
+    YouthApplicationViewSet.create_without_ssn endpoint.
+
+    These youth applications don't have a social security number or VTJ data.
+    """
+
+    non_vtj_birthdate = factory.Faker("date_of_birth", minimum_age=1, maximum_age=99)
+    non_vtj_home_municipality = factory.Faker(
+        "random_element",
+        elements=[
+            "Helsinki",
+            "Espoo",
+            "Vantaa",
+            "Utsjoki",
+            "Stockholm",
+            "Tallinn",
+            "",  # Optional field
+        ],
+    )
+    is_unlisted_school = True
+    social_security_number = ""
+    encrypted_original_vtj_json = None
+    encrypted_handler_vtj_json = None
+    additional_info_user_reasons = [AdditionalInfoUserReason.OTHER.value]
+    additional_info_description = factory.Faker("text", max_nb_chars=80)
+    creator = factory.SubFactory(HandlerUserFactory)
+    created_at = timezone.now()
+    modified_at = timezone.now()
+    receipt_confirmed_at = timezone.now()
+    additional_info_provided_at = timezone.now()
```

Comment:
Aren't these common for both types, so they could be moved to the model that is abstract for both? It would make more sense to only describe the fields that are some how modified from the base class and are different with the other models

Reply:
Also the modified_at is different in the other class
```python
class AbstractYouthApplicationFactory(factory.django.DjangoModelFactory):
    created_at = timezone.now()
    modified_at = factory.LazyAttribute(determine_modified_at)
```